CSS/CSS.sublime-syntax

```diff
@@ -567,7 +567,7 @@ contexts:
             )\b
       scope: support.constant.property-value.css
       # Generic Font Families: https://www.w3.org/TR/CSS2/fonts.html
-    - match: \b(?i:arial|century|comic|courier|garamond|georgia|helvetica|impact|lucida|symbol|system|tahoma|times|trebuchet|utopia|verdana|webdings|sans-serif|serif|monospace|fantasy|cursive)\b
+    - match: \b(?i:arial|century|comic|courier|garamond|georgia|helvetica|impact|lucida|symbol|system|tahoma|times|trebuchet|utopia|verdana|webdings|sans-serif|serif|monospace|fantasy|cursive)(?=[;,])\b
```

Comment:
We’ll want the lookahead to allow whitespace, and be followed by a `,`, `;`, `\n`, `}` or `)`. This will allow things like:

```css
.myclass { font-family: Helvetica Neue }
.myclass2 { font-family: Helvetica , Arial ; }
.myclass 3 {
    font-family: Arial
}
.myclass4 { font-family: var(—base-font, Arial) }
```